src/components/Footer/Footer.tsx

```diff
@@ -48,7 +48,7 @@ export function FooterSimple({ links }: FooterSimpleProps) {
   return (
     <div className={classes.footer}>
       <Container className={classes.inner}>
-        <Image src="./nist.png" height={105} fit="contain" />
+        <Image src={`${import.meta.env.BASE_URL}/nist.png`} height={105} fit="contain" />
```

Comment:
Non-blocking: I am not sure this was meant to fix the `nist.png` footer issue, but if this code is deployed and running for the current API Gateway instance, I still get issues with that.

Reply:
It's not currently running on the APIGW instance (good point, I'll test that now), but this is so when deployed on APIGW it looks for the logo on `/dev/nist.png` instead of `/nist.png`

Reply:
See #29